internal/core/metadata/rest_addressable.go

```diff
@@ -333,7 +333,7 @@ func restGetAddressableByAddress(w http.ResponseWriter, r *http.Request) {
 func notifyAddressableAssociates(a models.Addressable, action string) error {
 	// Get the devices
 	var d []models.Device
-	if err := dbClient.GetDevicesByAddressableId(&d, a.Id); err != nil {
```

Comment:
Need to keep `d` here for use on line 345

Reply:
Good catch.  I think I initially moved the variable to the left side of the declaration and the IDE told me it was an unused variable.  Assignment to a previously declared variable within an if initialization statement requires the variable be used within the body of the if.  Compiler throws an error if it isn't.  I'll refactor.